### PR TITLE
DOC-9588for6.6: Added info on group correspondences

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Comment by tonyjhillman** [comment]
Approved by Timofey at https://github.com/couchbase/docs-server/pull/2392

**Comment by tonyjhillman** [comment]
Approved by Timofey at https://github.com/couchbase/docs-server/pull/2392

File: modules/manage/pages/manage-security/configure-ldap.adoc [hunk 1]
```diff
@@ -380,6 +380,8 @@ Additionally, the *Clear Cache* button is provided, so that all currently cached
 
 To map an LDAP group to Couchbase-Server roles, create a Couchbase-Server user-group; associate the user-group with the LDAP group; and then assign roles to the user-group.
 
+(Note that each Couchbase-Server user-group can be associated with at most _one_ LDAP group &#8212; although if that LDAP group itself incorporates multiple additional LDAP groups by means of _LDAP-group nesting_, an _indirect_ association with those LDAP groups occurs.)
```
Comment (RayOffiah) [inline modules/manage/pages/manage-security/configure-ldap.adoc]:
```suggestion
(Note that each Couchbase-Server user-group can be associated with at most _one_ LDAP group -- although if that LDAP group itself incorporates multiple additional LDAP groups by means of _LDAP-group nesting_, an _indirect_ association with those LDAP groups occurs.)
```

**Review by RayOffiah — APPROVED** [review]
Small suggestion.

**Pull request merged**